src/crewai/agent.py

```diff
@@ -114,6 +114,14 @@ class Agent(BaseAgent):
         default=None,
         description="Embedder configuration for the agent.",
     )
+    agent_knowledge_context: Optional[str] = Field(
+        default=None,
+        description="Knowledge context for the agent.",
+    )
+    crew_knowledge_context: Optional[str] = Field(
```

Comment:
why does this attributes belongs to the agent instead of the crew?

Reply:
gotcha.. 
renaming would be good idea to improve clarity and readabiliy - not for now.. unfurtunely, I don't have any further suggestions right now. we can handle that later

Reply:
deal